lib/moonshot/stack.rb

```diff
@@ -208,7 +208,7 @@ def get_stack(name)
     def upload_template_to_s3
       raise 'The S3 bucket to store the template in is not configured.' unless @config.template_s3_bucket
 
-      s3_object_key = "#{@name}-#{Time.now.getutc.to_i}-#{File.basename(template.filename)}"
+      s3_object_key = "#{File.basename(template.filename)}"
```

Comment:
@name here is deployment name right?  losing that means multiple deployments will use the same path and conflict with each other.

Reply:
its even better if the path is bucket/app-name/env-name/filename